numeral.js

```diff
@@ -598,6 +617,10 @@
             return unformatNumeral(this, inputString ? inputString : defaultFormat);
         },
 
+        units : function () {
```

Comment:
I thought it when I wrote it, but decided against it for no specific reason. Since you had the same thought, I agree.